packages/components/organisms/f-cookie-banner/f-cookie-banner-static/gulpfile.js

```diff
@@ -0,0 +1,91 @@
+const { src, dest, series } = require('gulp');
+const log = require('fancy-log');
+const concat = require('gulp-concat');
+const rename = require('gulp-rename');
+const del = require('del');
+const { exec } = require('child_process');
+
+const LOCALES = ['en-GB', 'es-ES', 'it-IT', 'en-IE', 'en-AU', 'en-NZ'];
+
+const PATHS = {
+    vueSrcFolder: './src',
+    distFolder: './.tmp-dist',
+    tempVueFolder: './.tmp-vue'
+};
+
+const replace = require('gulp-replace');
+
+function setVueProps(locale) {
```

Comment:
May be worth adding some JSDoc comments to these functions, just to explain what each does